helm_deploy/hmpps-mailbox-register-api/values.yaml

```diff
@@ -1,19 +1,39 @@
+---
+# Default values.
+# Declare variables to be passed into your templates.
+#
+# Variables in `helm_deploy/values-[env].yaml` will take precedence.
+
 generic-service:
   nameOverride: hmpps-mailbox-register-api
-  productId: "DPS087" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
+  productId: DPS087 # TODO: Update with correct product ID
```

Comment:
Does this need to be done before this PR?

Reply:
Nope, will be done at a later point, the product needs to be created in the developer portal.
Andy suggested using this product Id for now.
There are a few placeholder / temp values for the time being, this is one 😄

Reply:
This will be needed, as it links the service to the team in the developer portal